charts/cluster/values.yaml

```diff
@@ -0,0 +1,211 @@
+  # Please refer to https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ for more information.
+  # We strongly advise you use the same setting for limits and requests so that your cluster pods are given a Guaranteed QoS.
+  # See: https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/
+  resources:
+    # limits:
+    #   cpu: 2000m
+    #   memory: 8Gi
+    # requests:
+    #   cpu: 2000m
+    #   memory: 8Gi
+
+  priorityClassName: "" # TODO: Blocked by https://github.com/cloudnative-pg/cloudnative-pg/discussions/1609
+
+  # Method to follow to upgrade the primary server during a rolling update procedure, after all replicas have been
+  # successfully updated. It can be switchover (default) or in-place (restart).
+  primaryUpdateMethod: switchover
+
+  # Strategy to follow to upgrade the primary server during a rolling update procedure, after all replicas have been
+  # successfully updated: it can be automated (unsupervised - default) or manual (supervised)
+  primaryUpdateStrategy: unsupervised
+
+  # The instances' log level, one of the following values: error, warning, info (default), debug, trace
+  logLevel: "info"
+
+  # Affinity/Anti-affinity rules for Pods
+  # See: https://cloudnative-pg.io/documentation/current/api_reference/#AffinityConfiguration
+  affinity:
+    topologyKey: topology.kubernetes.io/zone
+
+  # The configuration for the CA and related certificates
+  # See: https://cloudnative-pg.io/documentation/current/api_reference/#CertificatesConfiguration
+  certificates:
+
+  # When this option is enabled, the operator will use the SuperuserSecret to update the postgres user password.
+  # If the secret is not present, the operator will automatically create one.
+  # When this option is disabled, the operator will ignore the SuperuserSecret content, delete it when automatically created,
+  # and then blank the password of the postgres user by setting it to NULL.
+  enableSuperuserAccess: true
+  superuserSecret: ""
+
+  monitoring:
+    enablePodMonitor: true
+    grafanaDashboard:
+        enable: true # TODO
+    customQueries: []
+      # - name: "pg_cache_hit_ratio"
+      #   query: "SELECT current_database() as datname, sum(heap_blks_hit) / (sum(heap_blks_hit) + sum(heap_blks_read)) as ratio FROM pg_statio_user_tables;"
+      #   metrics:
+      #     - datname:
+      #         usage: "LABEL"
+      #         description: "Name of the database database"
+      #     - ratio:
+      #         usage: GAUGE
+      #         description: "Cache hit ratio"
+
+  # Configuration of the PostgreSQL server
+  # See: https://cloudnative-pg.io/documentation/current/api_reference/#PostgresConfiguration
+  postgresql:
+
+  # SQL Queries run during the initialization of the cluster
+  # Can be used to create databases, users, extensions, etc.
+  initSQL: []
+
+  additionalLabels: {}
+  annotations: {}
+
+###
+# Database cluster backup configuration
+backups:
+  enabled: true
```

Comment:
enabling backups by default means that by default it will not work with the following error:
```
Error: execution error at (cluster/templates/cluster.yaml:75:5): You need to specify S3 region if endpointURL is not specified.
```
I would probably say this should be false and we should warn in the ` NOTES.txt` that it's highly discouraged running without backups, if `backups.enabled` is `false`

Reply:
I'd like to always have sane defaults for a safe, high-availability setup in the chart. I'd rather do it the other way around - a more descriptive error, when all fields are missing, that says:

```txt
A backup storage method hasn't been configured. You can disable backups if not needed using the `backups.enabled` flag.
```

Reply:
Would that be sufficient?

Reply:
It depends on the goal of the chart, if it's just for production clusters then it's ok to fail at first and make the message clearer and I would make it clearer, even through the name of the chart, if it should instead be a more handheld experience to spinning up a Cluster at first locally and then up to production, I'd allow an unsafe config by default and add warnings

Reply:
I've went with your suggestion and have disabled backups by default. It's also backed by a warning message in the NOTES.txt. See the image below.